site/components/header/small-screen-nav.js

```diff
@@ -14,31 +14,36 @@ const trackAnalytics = title => () =>
     label: `From: ${window.location.pathname}`,
   });
 
-export default class SmallScreenNav extends React.Component {
+type State = {
+  navOpen: boolean,
+};
+
+export default class SmallScreenNav extends React.Component<any, State> {
   constructor() {
     super();
     this.state = {
       navOpen: false,
     };
   }
 
-  state: {
-    navOpen: boolean,
-  };
-  smallScreenNav: HTMLInputElement;
+  smallScreenNav: HTMLInputElement | HTMLDivElement;
 
   documentBodyLock = () => {
+    // $FlowIgnore
     document.getElementById('mainContent').setAttribute('aria-hidden', 'true');
+    // $FlowIgnore
     document.body.style.position = 'fixed';
     this.smallScreenNav.scrollTop = 0;
   };
 
   documentBodyRelease = () => {
+    // $FlowIgnore
     document.getElementById('mainContent').removeAttribute('aria-hidden');
+    // $FlowIgnore
     document.body.style.position = 'relative';
   };
```

Comment:
Think we can fix this without ignoring see https://github.com/redbadger/website-honestly/blob/circle-2.0/site/components/header/small-screen-nav.js#L31

Reply:
The problem is there are null checks and that's why it's falling over. Your file referenced does have those checks. I didn't want to edit any implementation and only the typings. Happy to go back and change?

Reply:
yup lets change it!

Reply:
Think there is no avoiding that really with the way the typings work. I'd rather fix it now to be honest.